PacketTunnelProvider/AppTrackingProtectionPacketTunnelProvider.swift

```diff
@@ -98,11 +109,12 @@ class AppTrackingProtectionPacketTunnelProvider: NEPacketTunnelProvider {
     }
     
     override func sleep(completionHandler: @escaping () -> Void) {
-        // Add code here to get ready to sleep.
-        completionHandler()
+        Pixel.fire(pixel: .appTPVPNWake) { _ in
+            completionHandler()
+        }
     }
```

Comment:
I'm not sure that this function is being called as we don't have the appropriate flag set, but I'd like to be sure about that, so let's monitor it.